server/views/pages/offence/check-offence-answers.njk

```diff
@@ -105,19 +74,84 @@
                 }}
             {% endfor %}
 
+            {% if courtAppearance.offences.length > 0 %}
             <div class="govuk-button-group govuk-!-margin-top-8">
                 <form method="post" action="/person/{{ nomsId }}/{{ addOrEditCourtCase }}/{{ courtCaseReference }}/{{ addOrEditCourtAppearance }}/{{ appearanceReference }}/submit-check-offence-answers">
                     <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+                    {{ govukRadios({
+                        id: "finishedAddingOffences",
+                        name: "finishedAddingOffences",
+                        fieldset: {
+                            legend: {
+                                text: "Have you finished adding offences?",
+                                classes: "govuk-fieldset__legend--m"
+                            }
+                        },
+                        errorMessage: errors | findError('finishedAddingOffences'),
+                        items: [
+                            {
+                                value: "true",
+                                text: "Yes, I've finished adding offences",
+                                attributes: {
+                                "data-qa": "finishAddingRadio"
+                                }
+                            },
+                            {
+                                value: "false",
+                                text: "No, I'll come back later",
+                                attributes: {
+                                "data-qa": "notFinishedRadio"
+                                }
+                            }
+                        ]
+                    }) }}
+
                     {{ govukButton({
-                        text: "Accept and continue",
+                        text: "Continue",
                         attributes: {
                             "data-qa": "finishAddingButton"
                         }
                     }) }}
             </form>
             </div>
+            {% endif %}
 
         </div>
+        <div class="govuk-grid-column-one-third">
```

Comment:
There is a partial for appearance details layout which this page now should extend https://github.com/ministryofjustice/hmpps-remand-and-sentencing/blob/main/server/views/partials/appearanceDetailsLayout.njk (example of a view which uses it https://github.com/ministryofjustice/hmpps-remand-and-sentencing/blob/main/server/views/pages/offence/offence-code.njk)

Reply:
Nice, updated to make use of that